### ISSUE TITLE
Use yarn to install beta version have incompatible problem

### ISSUE THREAD

**Issue by zack9433** [angular-ui/ui-router#3086]
My yarn version is v0.15.1 and node version is v4.x

There is an error occur
```
error angular-ui-router@1.0.0-beta.1: The engine "node" is incompatible with this module. Expected version ">4".
error Found incompatible module
```

I thinks yarn will look into `engines` version in package.json file.

**Comment by christopherthielen** [comment]
Thanks, I'll take a look

**Comment by samithaf** [comment]
Looks like NPM also catches this one but just log it as a Warn rather than an error.

> 10:00:39.981 npm WARN engine angular-ui-router@1.0.0-alpha.5: wanted: {"node":">4"} (current: {"node":"4.4.7","npm":"2.15.8"})

**Comment by Luddinus** [comment]
+1

**Comment by hulkish** [comment]
Is there amy chance we can get this fix published to npm? id like to start using yarn for my project and this seems to.be blocking at the moment

**Comment by Luddinus** [comment]
This is the message I'm getting: (windows 7), if it helps

"error angular-ui-router@1.0.0-beta.3: The engine "node" is incompatible with this module. Expected version ">4".
error Found incompatible module"

**Comment by jonrimmer** [comment]
As a workaround, Yarn has an "--ignore-engines" CLI flag that should skip the check. [You'll need to be running Yarn v0.16](https://github.com/yarnpkg/yarn/issues/972).

**Comment by christopherthielen** [comment]
I'd like to get beta.4 out ASAP, but there are a few things blocking it at the moment:

### Split mono-repo
core, ng1 and ng2  are now split out as three separate repositories.  This housekeeping is mostly done.  I think the last things to to address are:

- how to do the bower publishing for ng1.
- commits to `core` should cause CI to fetch the other repos and run the unit tests (for `ng1`, `ng2`, and `react`)  -- this doesn't have to be complete before a beta.4 release

### ng1/ng2 docs split 
I've split up the generated API documentation into ng1 and ng2.  These need to be published to the `ui-router.github.io` site in a way that doesn't break most of the existing docs links.  

Instead of `/docs/latest` and `/docs/1.0.0-beta.3` I think we'll do `/ng1/docs/latest` and `/ng2/docs/1.0.0-beta.4`

When I moved docs earlier I used redirect files.  This will probably work for this scenario too.

### ng1 docs simplification
I've simplified the docs for ng2, hiding most of the internal apis.
The same should be done for ng1 docs.

This isn't necessarily a blocker for beta.4 but would be nice to have.
